lib/gooddata/lcm/lcm2.rb

```diff
@@ -35,6 +36,19 @@ def method_missing(name, *_args)
         end
       end
 
+      def [](variable)
```

Comment:
this is a good change in my opinion, but I wouldn't bundle it with this pull request as it is a different problem altogether